src/app/Common/DAL/Documents/Token/Token.cs

```diff
@@ -9,5 +10,8 @@ public class Token : BaseDocument
         public string Value { get; set; }
         public DateTime ExpireAt { get; set; }
         public string UserId { get; set; }
+        
+        [BsonIgnore]
+        public bool IsExpired => ExpireAt <= DateTime.UtcNow;
```

Comment:
At TheraNest we try to get rid of computed properties inside of DAL classes as it can cause serialization/deserialization problems. Good and real example is caching.

I'm not sure it's can be applied to this project but it's definitely worth mentioning.

If you decided to fix that please consider putting this computed thing either to extension or to service class.